File: AUGUST-2025/14-08-2025/p2264.php
class Solution {

    /**
     * @param String $num
     * @return String
     */
    function largestGoodInteger($num) {
        $codes = ['999', '888', '777', '666', '555', '444', '333', '222', '111', '000'];

foreach ($codes as $code) {
    if (strpos($num, $code) !== false) {
        return  $code;
    }
        
    }
    return ''; 
    }
}

File: AUGUST-2025/14-08-2025/p2264.scala
object Solution {
    def largestGoodInteger(num: String): String = {
        num.toCharArray.sliding(3).filter {
            case Array(v1, v2, v3) => v1 == v2 && v1 == v3
        }.map(_.mkString).maxOption.getOrElse("")
    }
}